includes/components/class-admin.php

```diff
@@ -1867,4 +1904,30 @@ function( $config ) {
 			]
 		);
 	}
+
+	/**
+	 * Update dismissed HivePress pointers.
+	 */
+	public function update_dismiss_pointers() {
```

Comment:
is there a way to use the controller instead, or it's a common API for tooltips?